test/Table.test.tsx

```diff
@@ -61,6 +61,17 @@ const ref = React.createRef<TableInstance<Row, string>>();
 
 <Table ref={ref} />;
 
+<Table ref={ref}>
+  {({ Column, HeaderCell, Cell }) => (
+    <>
+      <Column>
+        <HeaderCell>Id</HeaderCell>
+        <Cell>{({ id }) => id}</Cell>
+      </Column>
+    </>
+  )}
+</Table>;
+
```

Comment:
It is recommended to add a type check.

```suggestion
interface InventoryItem {
  id: string;
  name: string;
}

const table = React.createRef<TableInstance<InventoryItem, string>>();

<Table<InventoryItem, string> ref={table}>
  {({ Column, HeaderCell, Cell }) => (
    <>
      <Column>
        <HeaderCell>Name</HeaderCell>
        <Cell>{row => row.name}</Cell>
      </Column>
      <Column>
        <HeaderCell>Type</HeaderCell>
        {/** @ts-expect-error Property 'type' does not exist on type 'InventoryItem' */}
        <Cell>{row => row.type}</Cell>
      </Column>
    </>
  )}
</Table>;
```